examples/instantsearch+helper.html

```diff
@@ -43,13 +44,13 @@
           disjunctiveFacets: ['category', 'manufacturer'],
           hitsPerPage: 5,
           maxValuesPerFacet: 5,
-          getRankingInfo : true
+          getRankingInfo: true
         });
 
-        helper.on( "result", searchCallback);
+        helper.on("result", searchCallback);
 
-        $inputfield.keyup( function( e ){
-          helper.setQuery( $inputfield.val() ).search();
+        $inputfield.keyup(function(e){
+          helper.setQuery($inputfield.val()).search();
         } );
```

Comment:
Why do we have space at the end of a pair of parentheses and not at the beginning?